src/explore-education-statistics-admin/src/pages/publication/components/ReleaseContributorPermissions.tsx

```diff
@@ -0,0 +1,68 @@
+import { ManageAccessPageContributor } from '@admin/services/releasePermissionService';
+import ButtonText from '@common/components/ButtonText';
+import ModalConfirm from '@common/components/ModalConfirm';
+import WarningMessage from '@common/components/WarningMessage';
+import styles from '@admin/pages/publication/components/ReleaseContributorPermissions.module.scss';
+import React, { useState } from 'react';
+
+export interface Props {
+  contributors: ManageAccessPageContributor[];
+  handleUserRemoval: (userId: string) => void;
+}
+
+const ReleaseContributorPermissions = ({
+  contributors,
+  handleUserRemoval,
+}: Props) => {
+  const [removeUser, setRemoveUser] = useState<ManageAccessPageContributor>();
+
+  return (
+    <>
+      {!contributors || contributors.length <= 0 ? (
```

Comment:
An array's length should never be less than 0.

Reply:
I had a university lecturer who said you should use less than in these circumstances - it struck me as a little odd even back then, but I somehow fell into the habit of doing it.